app/views/homes/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if @top_work %>
+<h4>Media Spotlight: <%= link_to(@top_work.title, work_path(@top_work.id)) %> by <%= @top_work.creator%></h4>
+
+<h6><%= @top_work.vote_count %> votes - <%= @top_work.description%></h6><br>
+<%  end %>
+
+
+<h5>Top Movies</h5>
+<ul>
```

Comment:
You should be using some semantic HTML here! Each of these pieces (top work, albums, books, etc) could be split into a separate `<section>`